### DISCUSSION TITLE
Feature request: Input parameters in odf-header

### DISCUSSION THREAD

**Discussion by wsc44** [GrandOrgue/OdfEdit#52] [Ideas]
Please print OdfEdit-input parameters into file header data (example: "; Do not convert noises: Yes / Convert alternate Panels: No / ..."). That feature would be very helpful when comparing various odf-versions, expecially when generated months ago.

**Comment by eturpault** [comment]
Good idea, I will add this in next OdfEdit version.

**Comment by eturpault** [comment]
This has been implemented in OdfEdit v2.12.
